code/modules/mob/mob_grab_specials.dm

```diff
@@ -35,6 +35,10 @@
 		if(H.getOxyLoss() >= 20)
 			to_chat(user, "<span class='warning'>[H]'s skin is unusually pale.</span>")
 			bad = 1
+		if(E.is_infected())
+			var/severity = E.germ_level < INFECTION_LEVEL_TWO ? "slightly" : E.germ_level < INFECTION_LEVEL_THREE ? "moderately" : "extremely"
```

Comment:
cooking the mother of all ternarnies, jack

Reply:
I just think they're neat.